metrics/web/docs/features/firestore_emulator_for_driver_tests/diagrams/firestore_emulator_sequence_diagram.puml

```diff
@@ -0,0 +1,42 @@
+@startuml firestore_emulator_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant FlutterWebDriver
+participant ProcessManager
+participant FirestoreEmulatorRunner
+participant FirestoreEmulatorProcess
+participant FlutterDriveProcessRunner
+
+Bob -> FlutterWebDriver : runs tests
+activate FlutterWebDriver
+
+FlutterWebDriver -> ProcessManager : creates
+activate ProcessManager
+deactivate FlutterWebDriver
+
+ProcessManager -> FirestoreEmulatorRunner : run()
+activate FirestoreEmulatorRunner
+
+
+FirestoreEmulatorRunner -> FirestoreEmulatorProcess : start()
+activate FirestoreEmulatorProcess
+deactivate FirestoreEmulatorRunner
+
+FirestoreEmulatorProcess -> FirestoreEmulatorProcess : runs the emulator (firebase emulators:start --only firestore)
```

Comment:
Would it be great to mention that the `ProcessManager` waits until the Firestore emulator become available before running the driver tests?